resources/views/platform/completion-progress-page.blade.php

```diff
@@ -0,0 +1,86 @@
+<?php
+use App\Site\Models\User;
+
+$targetUsername = $user->User;
+$isMe = $me->User === $targetUsername;
+
+$headingLabel = '';
+if ($isMe) {
+    $headingLabel = 'Your Completion Progress';
+} elseif (substr($targetUsername, -1) === 's') {
+    $headingLabel = $targetUsername . "' Completion Progress";
+} else {
+    $headingLabel = $targetUsername . "'s Completion Progress";
+}
+
+// TODO: Once using denormalized data, come up with a good page description.
+// Doesn't matter right now because these pages don't generate any SEO juice.
+// $pageDescription = "";
+?>
+
+<x-app-layout :pageTitle="$headingLabel">
+    <div>
+        <x-completion-progress-page.breadcrumbs :targetUsername="$targetUsername" />
+
+        <div class="mt-3 -mb-3 w-full flex gap-x-3">
+            {!! userAvatar($targetUsername, label: false, iconSize: 48, iconClass: 'rounded-sm') !!}
+            <h1 class="mt-[10px] w-full">{{ $headingLabel }}</h1>
+        </div>
+
+        <x-completion-progress-page.meta-panel
+            :availableConsoleIds="$allAvailableConsoleIds"
+            :selectedConsoleId="$selectedConsoleId"
+            :selectedStatus="$selectedStatus"
+        />
+
+        <div class="mb-4">
+            <x-completion-progress-page.awards-jumbobox
+                :playedCount="$primaryCountsMetrics['numPlayed']"
+                :unfinishedCount="$primaryCountsMetrics['numUnfinished']"
+                :beatenSoftcoreCount="$primaryCountsMetrics['numBeatenSoftcore']"
+                :beatenHardcoreCount="$primaryCountsMetrics['numBeatenHardcore']"
+                :completedCount="$primaryCountsMetrics['numCompleted']"
+                :masteredCount="$primaryCountsMetrics['numMastered']"
+            />
+        </div>
+
+        @if ($totalInList !== $primaryCountsMetrics['numPlayed'])
+            @if ($totalInList > 0)
+                <p class="text-xs mb-2">
+                    Viewing
+                    <span class="font-bold">{{ localized_number($totalInList) }}</span>
+                    @if ($isFiltering)
+                        of {{ $primaryCountsMetrics['numPlayed' ]}}
+                    @endif
+                    {{ trans_choice(__('resource.game.title'), $isFiltering ? $primaryCountsMetrics['numPlayed'] : $totalInList) }}
+                </p>
+            @endif
+        @elseif ($totalInList === 0)
```

Comment:
I see a message when there are no matching games for the current filter for users with no activity but I do not see that message when clicking someone that has something like 0 games beaten and therefore no matching games when clicked.

Ex. No achievements earned
![image](https://github.com/RetroAchievements/RAWeb/assets/4047771/df57efe6-39a4-423a-9b0d-e5e22674ceee)

Ex. Clicking 0 Beaten for someone with achievements earned
![image](https://github.com/RetroAchievements/RAWeb/assets/4047771/1aea016b-7dba-4f89-af14-8f6bf36f2303)

Reply:
Looks like I introduced some flawed conditional logic in a refactor, should be resolved as of https://github.com/RetroAchievements/RAWeb/pull/1833/commits/8279fc6f0cdd593e76cea210bd7eaae2e42a1465.